pandas/tests/test_window.py

```diff
@@ -1228,242 +1170,75 @@ def roll_mean(x, window, min_periods=None, freq=None, center=False,
 
     def test_rolling_apply_out_of_bounds(self):
         # #1850
-        arr = np.arange(4)
+        vals = pd.Series([1, 2, 3, 4])
 
-        # it works!
-        with catch_warnings(record=True):
-            result = mom.rolling_apply(arr, 10, np.sum)
-        assert isna(result).all()
+        result = vals.rolling(10).apply(np.sum)
+        assert result.isna().all()
 
-        with catch_warnings(record=True):
-            result = mom.rolling_apply(arr, 10, np.sum, min_periods=1)
-        tm.assert_almost_equal(result, result)
+        result = vals.rolling(10, min_periods=1).apply(np.sum)
+        expected = pd.Series([1, 3, 6, 10], dtype=float)
+        tm.assert_almost_equal(result, expected)
 
     def test_rolling_std(self):
-        self._check_moment_func(mom.rolling_std, lambda x: np.std(x, ddof=1),
+        self._check_moment_func(lambda x: np.std(x, ddof=1),
                                 name='std')
-        self._check_moment_func(mom.rolling_std, lambda x: np.std(x, ddof=0),
+        self._check_moment_func(lambda x: np.std(x, ddof=0),
                                 name='std', ddof=0)
 
     def test_rolling_std_1obs(self):
-        with catch_warnings(record=True):
-            result = mom.rolling_std(np.array([1., 2., 3., 4., 5.]),
-                                     1, min_periods=1)
-        expected = np.array([np.nan] * 5)
-        tm.assert_almost_equal(result, expected)
```

Comment:
there are 2 tests here (diff ddof)

Reply:
Not sure I understand, I'm replicating what was there before.

Reply:
ahh I c, ok then.